src/components/buttons/HdRadioButton.vue

```diff
@@ -52,7 +52,7 @@ export default {
   props: {
     label: String,
     value: {
-      type: [String, Number],
+      type: [String, Number, Object],
```

Comment:
Could you link sliders PR here so we can better understand the use case?

Reply:
This PR, where the value I need is the option selected:
https://github.com/homeday-de/sliders/pull/1193
I could use the index of the option and then reference the option with the position at the array, but I don't see why not just pass the object.